common/customizable_localization/wc_lifestyles_custom_loc.txt

```diff
@@ -2,48 +2,48 @@
 	type = character
 
```

Comment:
Prowess traits need to be replaced by our physical traits. They represent the same.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/155839422-25a3a767-05b8-4b59-bd43-16287045dea1.png)

</details>

Reply:
Dunno, I wouldn't say that our physical traits and trait indicating knowledge about character's cultural traditional combat represent the same thing.